CMSgov/design-system
# [WNMGDS-2893] Implement Tooltip web component

## Summary

- Initial implementation of the ds-tooltip web component
- Stories are written, and seek to emulate the existing React-based component
- Tests are written, and seek to emulate the existing React-based component
- There are a few notable differences between the web component and the React component:
  - No onOpen or onClose custom events - I figured these could be manually attached rather than being inbuilt, especially since these are used somewhat infrequently and for analytics when they are used.
  - No aria-label - see [Slack Thread](https://cmsgov.slack.com/archives/CT169N26L/p1727468064914539) for context. 

## How to test

1. Pull down branch
2. Confirm tests pass locally
3. Confirm controls in stories all change the component in expected ways.
4. Confirm that the usability of the component matches the expectations from the React component.

## Checklist
